dcs_deploy.py

```diff
@@ -706,10 +868,13 @@ def generate_images(self):
                 else:
                     opt_app_size = 8
                 opt_app_size_arg = f"-S {opt_app_size}GiB"
-                external_only = "" # flash internal and external device
-                #self.rootdev = "external" # set UUID device in kernel commandline: rootfs=PARTUUID=<external-uuid>
+                external_only = "" # Flash internal and external device
+
+            if self.args.app_size is not None:
+                opt_app_size_arg = f"-S {self.args.app_size}GiB"
+
```

Comment:
Maybe use some default size?

Reply:
This is from nvidia `l4t_initrd_flash.func`:
```
    -S <size>                    External APP partition size in bytes. KiB, MiB, GiB short hands are allowed,
                                 for example, 1GiB means 1024 * 1024 * 1024 bytes. (optional)
```

It is marked as optional. Would it still make sense to set default APPSIZE for you?

Reply:
It is ok like this then.